packages/noports_core/lib/src/sshnp/util/ssh_session_handler/dart_ssh_session_handler.dart

```diff
@@ -146,7 +166,7 @@ class SshClientHelper {
     }
   }
 
-  Future<void> startForwarding(
+  Future<int> startForwarding(
```

Comment:
if fLocalPort is 0, then an ephemeral port is generated and we need to return it